### PR TITLE
Integration of new layout with new pinia store

### LINKED ISSUE #152: Wallet | Menu and Account info and overflow menu
## Overview
As a user, I want the main view of the Telos EVM Wallet to display my account address and have a copy button, so that I can easily share it with others. I also want to be able to log out and go to Teloscan.

## Acceptance Criteria
- display account address 
- display copy button
    - copy button when pressed copies address to clipboard
- display overflow menu containing:
    - logout
    - external link to Teloscan
- display main menu with links to:  
  - Open Native Wallet link with icon that opens landing page with Native tab selected
  - Wallet and Sign Out
  - Include icons as on figma design
  - Sign out should navigate user to Landing Page and disconnect the account from application and allow user to connect a evm wallet again.

### Pre-requisites
- [ ] #168
- [ ] #169

### Components
- [ ] #170
- [ ] #171

image

### LINKED ISSUE #205: integrate EVM hyperion API from teloscan into Antelope lib
## Overview
Currently, the Antelope library does not have any methods to query data from EVM blockchain  (such as account balances and transaction history). Therefore, we need to integrate into the Antelope library, the existing code from Teloscan for interacting with the Hyperion API.

## Stories
- As a developer, I want to have a set of functions (an API) within the Antelope library that allows me to query data from the EVM using the Hyperion API.
- As a developer, I want the API to be well-organized and easy to use, so that I can quickly and easily access the information I need.

## Acceptance Criteria
- The Antelope library must have a set of functions that allow developers to query data from the EVM using the Hyperion API.
- All the configurations needed to set the connection should be present in the EVM chain instance.
- The API should include functions for querying account balances and transaction history, as these are the most common use cases.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #152, #170, #171
## Description
This PR integrates the latest visual components refactoring work (from [PR218](https://github.com/telosnetwork/telos-wallet/pull/218) and [PR193](https://github.com/telosnetwork/telos-wallet/pull/193) @EJSG's work) on the EVM wallet's user interface, along with the new implementation of the Pinia store-based library (from [PR219](https://github.com/telosnetwork/telos-wallet/pull/219) and [PR211](https://github.com/telosnetwork/telos-wallet/pull/211)).

The site's navigation now includes user authentication, along with some basic functionalities such as displaying the logged-in account's address, copying it to the clipboard with a click, and displaying a menu with options to redirect to Telosscan, log out, or switch to dark mode.

As for the native part, there shouldn't be any changes beyond the new way of authenticating from the new HomePage.

**Note:** since [PR219](https://github.com/telosnetwork/telos-wallet/pull/219) is still not merged but is included in this PR I will close PR219 and include the issues index in this PR

## Test scenarios
- go to [this link](https://deploy-preview-220--wallet-staging.netlify.app/)
- press **Connect your wallet button**
  - The MetaMask wallet should popup
  - if you don't have the network configured, it will prompt you to add it
  - if you are not connected to that chain it will ask you to change the current chain
  - if you are not authenticated you will be asked to authenticate
  - If you finally log in, you will be redirected to `/evm/wallet?tab=balance` (**new App Skeleton**)
- once logged in, you should see your real address displayed on the top-right
- if you press the button next to the address
  - it should copy the complete address to your clipboard
  - A notification should appear saying that
- If you press the account dropdown button, a menu should appear with three options:
  - teloscan: which redirects you to a teloscan page of your logged account address
  - logout: which actually logs you out and redirects you to the home page
  - dark mode: which switched between dark and light modes 
- autologin: this works automatically and should redirect you to the correct page
  - if you are logged in, go to [homepage](https://deploy-preview-220--wallet-staging.netlify.app/)
     - it should redirect you to  `/evm/wallet?tab=balance` 
  - if you are logged out, go to any `/evm/...` path link [this link](https://deploy-preview-220--wallet-staging.netlify.app/evm/wallet?tab=balance) or `[this link](https://deploy-preview-220--wallet-staging.netlify.app/evm/wrap?tab=unwrap)`
    - then you should be redirected to the Home page

## Checklist:
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file
-   [x] I have created a new issue with the required translations for the currently supported languages
-   [ ] I have added appropriate test coverage

### CODE REVIEW

File: src/components/evm/UserInfo.vue [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                class="c-user-info__dropdown-item"
+                role="menuitem"
+                tabindex="2"
+                @click="toggleDarkMode"
+            >
+                <InlineSvg
+                    :src="require('src/assets/icon--logout.svg')"
+                    :class="{
+                        'c-user-info__icon': true,
+                        'c-user-info__icon--hide': true,
```
[the remark this answers is not in the document]
Comment (Viterbo) [reply src/components/evm/UserInfo.vue]: Yes we can ([e007fad](https://github.com/telosnetwork/telos-wallet/pull/220/commits/e007fad47e75a0dcc30874d2b6debe14b0c2ff43))

File: src/components/evm/UserInfo.vue [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+export default defineComponent({
+    name: 'UserInfo',
+    components: {
+        InlineSvg,
+    },
+    methods: {
+        copyAddress() {
+            if (!accountStore.loggedAccount.isNative && accountStore.isAuthenticated) {
+                const address = accountStore.loggedEvmAccount.account;
+                navigator.clipboard.writeText(address.toUpperCase());
```
Comment (EJSG) [inline src/components/evm/UserInfo.vue:18]: this should actually be checksum, not uppercase

File: src/components/evm/UserInfo.vue [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                return;
+            } else {
+                ant.config.notifyErrorHandler(this.$t('settings.no_explorer', { network: accountStore.loggedAccount.network }));
+            }
+        },
+    },
+    computed: {
+        address() {
+            if (accountStore.isAuthenticated) {
+                return accountStore.loggedEvmAccount.displayAddress;
```
Comment (EJSG) [inline src/components/evm/UserInfo.vue:41]: this should also be checksum not caps

File: src/components/evm/UserInfo.vue (hunk 2 above)
Comment (Viterbo) [reply src/components/evm/UserInfo.vue:18]: What do you mean by that? can you suggest the correct code?

File: src/components/evm/UserInfo.vue (hunk 3 above)
Comment (Viterbo) [reply src/components/evm/UserInfo.vue:41]: done ([6ed1f67](https://github.com/telosnetwork/telos-wallet/pull/220/commits/6ed1f6770f0b97f1fee1dc905517b08de7b3c59e))

**Review by EJSG — APPROVED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**